cli/args/flags.rs

```diff
@@ -1350,6 +1363,28 @@ Show documentation for runtime built-ins:
             .help("Output documentation in JSON format")
             .action(ArgAction::SetTrue),
         )
+        .arg(
+          Arg::new("html")
+            .long("html")
+            .help("Output documentation in HTML format")
+            .action(ArgAction::SetTrue)
+            .conflicts_with("json")
+        )
+        .arg(
+          Arg::new("name")
+            .long("name")
+            .help("The name that will be displayed in the docs")
+            .action(ArgAction::Set)
+            .required_if_eq("html", "true")
+            .require_equals(true)
+        )
+        .arg(
+          Arg::new("output")
```

Comment:
maybe parse & validate as PathBuf, and store a PathBuf in `DocHtmlFlag` instead of String?